content/en/docs/setup/install/gateway/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Installing Gateways
+description: Install and customize Istio Gateways.
+weight: 40
+keywords: [install,gateway,kubernetes]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+Along with creating a service mesh, Istio allows managing [gateways](/docs/concepts/traffic-management/#gateways),
+which run at the edge of the mesh and allow fine-grained control over traffic entering and leaving the mesh.
+
+Follow this guide to deploy one or more gateways.
+
+## Prerequisites
+
+This guide requires the Istio control plane [to be installed](/docs/setup/install/) before proceeding.
+
+## Installing the gateway with injection
+
+To support users that already utilize existing deployment tools, we provide a few different ways to deploy a gateway.
+Each method will result in the same outcome; we recommend choosing the method you are most familiar with.
+If you are not sure which to choose, we suggest the Simple YAML method.
+
+As a security best practice, it is recommended to deploy the gateway in different namespace from the control plane.
+
+Using the same mechanisms as [Istio sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection), Istio allows the required
+options for gateways to run to be automatically injected.
+This enables developers to have full control over the gateway deployment, while simplifying operations - when a new upgrade is available, or a configuration has changed, gateway pods can simply be restarted to be updated.
```

Comment:
What are the customizations that can be done currently when using auto gateway injection and how can users do that?

Reply:
Its just a standard Kubernetes pod/deployment. You should be able to set whatever options you want following Kubernetes docs (`kubectl explain pod`)